ee/management/commands/test/test_sync_replicated_schema.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from django.conf import settings
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.sql.schema import CREATE_TABLE_QUERIES, EVENTS_TABLE_SQL
+from ee.clickhouse.util import ClickhouseTestMixin
+from ee.management.commands.sync_replicated_schema import Command
+from posthog.test.base import BaseTest
+
+
+@pytest.mark.ee
+class Test0004ReplicatedSchema(BaseTest, ClickhouseTestMixin):
```

Comment:
non-blocking

Should this be called TestSyncReplicatedSchema, or is this specific to 0004 migration or something?